MdeModulePkg/Universal/SetupBrowserDxe/Presentation.c

```diff
@@ -1606,8 +1642,7 @@ ProcessUserInput (
   if (UserInput->Action != 0) {
     Status                       = ProcessAction (UserInput->Action, UserInput->DefaultId);
     gCurrentSelection->Statement = NULL;
-  } else {
-    ASSERT (Statement != NULL);

```

Comment:
We've lost an ASSERT here. I don't mean stick it back in where it can't happen, but rather:
```
} else if (Statement == NULL {
  ASSERT (Statement != NULL);
} else {
```

Reply:
adding a return EFI_NOT_FOUND after the recommended assert, for the reason:

ASSERT by itself is not enough to cover cases. if DebugAssertEnabled returns false (`PcdDebugPropertyMask has bit DEBUG_PROPERTY_DEBUG_ASSERT_ENABLED cleared) , code execution will proceed pass the assert

Reply:
Resolved ('Resolve comment' is failing with 'Failed to resolve thread' for me for some reason).